FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -8,22 +8,34 @@
 import SwiftUI
 
 struct HomeView: View {
+
+
+
+
+
+
     // MARK: - Properties
     @EnvironmentObject var quizManager: QuizManager
     @EnvironmentObject var navigationManager: NavigationManager
     @Binding var presentSideMenu: Bool
-    private let levelItem: [LevelItem] = [
+    private let levelItem: [LevelItem] =     [
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/0e74ba265c47132c04f133da09675f0b9e81baab/FlashEnglish/Sources/Views/Home/HomeView.swift#L21"></span>Operators should be surrounded by a single whitespace when they are being used<br />
<code>operator_usage_whitespace</code> <code>HomeView.swift:21</code></td>
    </tr>
  </tbody>
</table>